src/user/UserRouter.ts

```diff
@@ -6,6 +6,16 @@ const router = Router();
 router.post('/api/1.0/users', async (req: Request, res: Response) => {
   const requestBody = req.body as { password: string; username: string; email: string };
 
+  const { username } = requestBody;
+
+  if (username === null) {
+    return res.status(400).send({
```

Comment:
Will return validationError field for response, the return is there so that it won't continue to the UserService below